queries.sql

```diff
@@ -90,8 +113,17 @@ SELECT billing_state, COUNT(*), SUM(total), AVG(total) FROM invoices WHERE billi
 
 -- The count, total, and average of invoice totals, grouped by state, ordered by average invoice total from highest-to-lowest
 
+SELECT billing_state,COUNT(*), SUM(total), AVG(total) FROM invoices GROUP BY billing_state ORDER BY AVG(total) DESC;
+
 -- A list of the top 5 countries by number of invoices
 
+SELECT billing_country, COUNT(*) FROM invoices GROUP BY billing_country ORDER BY COUNT(*) DESC LIMIT 5;
+
+
 -- A list of the top 5 countries by gross/total invoice size
 
+SELECT billing_country, COUNT(*), SUM(total) FROM invoices GROUP BY billing_country ORDER BY COUNT(*)/SUM(total) DESC LIMIT 5;
```

Comment:
Hah, I meant "gross (aka total) invoice size", not literal division! You're sorting by `1/AVG(total)`, which is an unusual thing to sort by.